server/storage/interface_test.go

```diff
@@ -0,0 +1,32 @@
+package storage
+
+import (
+	"testing"
+
+	"github.com/docker/notary/tuf/data"
+	"github.com/stretchr/testify/require"
+)
+
+func ConsistentEmptyGetCurrentTest(t *testing.T, s MetaStore) {
+	_, byt, err := s.GetCurrent("testGUN", data.CanonicalRootRole)
+	require.Nil(t, byt)
+	require.Error(t, err, "There should be an error Getting an empty table")
+	require.IsType(t, ErrNotFound{}, err, "Should get a not found error")
+}
+
+func ConsistentMissingTSAndSnapGetCurrentTest(t *testing.T, s MetaStore) {
```

Comment:
Non-blocking - since this is specifically a feature of `TUFMetaStorage` and not part of the `MetaStore` interface docs, would it make sense for this function to take a `TUFMetaStorage` instead?  That way to make it more valid, it can check that the underlying `MetaStore` does indeed have root metadata first, but `TUFMetaStorage` would fail to return it because it's missing timestamp or snapshot?